src/resources/Sources/SourcesMetadata/SourcesMetadataInterfaces.ts

```diff
@@ -0,0 +1,18 @@
+export interface Metadata {
+    /**
+     * The key for a metadata.
+     */
+    key: string;
+    /**
+     * The frequency at which this metadata has been seen on documents.
+     */
+    frequency: number;
+    /**
+     * Subset of values for the metadata.
+     */
+    values: any[];
```

Comment:
Ça va avec l'autre PR, mais on pourrait dire qu'on est plus descriptif sur le modèle (sample values au lieu de values).

Reply:
Updated to `Subset of sample values for the metadata.`

Reply:
En fais je pensais plus au nom de la variable dans le modèle 😅